rust/scx_utils/src/lib.rs

```diff
@@ -53,3 +53,6 @@ pub use topology::Node;
 
 mod cpumask;
 pub use cpumask::Cpumask;
+
+mod load_balance;
+pub use load_balance::LoadBalancer;
```

Comment:
I  wonder whether the name is too broad. It's likely that scheduler implementations want to use the term load balancer for themselves and it can get confusing for utility create to occupy that name. Would it make sense to name it more narrowly? This is really just to solve the infeasible weight problem, right? Would it make sense to name it accordingly?

Reply:
Yeah, let's rename it to `infeasible_calc.rs` or `infeasible_weights.rs`. I'm leaning towards `load_calc.rs` just because it also sums loads that can be used for under-utilized systems, etc. But yeah LoadBalance is too broad (as evidenced by the fact that I had to change the object name in rusty).